src/Microsoft.ML.LightGbm/WrappedLightGbmBooster.cs

```diff
@@ -80,14 +80,14 @@ internal unsafe string GetModelString()
             byte[] buffer = new byte[bufLen];
             int size = 0;
             fixed (byte* ptr = buffer)
-                LightGbmInterfaceUtils.Check(WrappedLightGbmInterface.BoosterSaveModelToString(Handle, 0, BestIteration, bufLen, ref size, ptr));
+                LightGbmInterfaceUtils.Check(WrappedLightGbmInterface.BoosterSaveModelToString(Handle, 0, BestIteration, 0, bufLen, ref size, ptr));
```

Comment:
Was this argument mismatch the source of the Internal CLR Error you were seeing last week?

Reply:
Nope. That was a new thing added in to the new versions that we aren't using at all, but it didn't exist in the prior version. I did figure out the Internal CLR issue though just barely. :) so just waiting on builds now.